core/domain/platform_parameter_domain_test.py

```diff
@@ -132,22 +143,24 @@ def test_is_valid_with_valid_android_context_returns_true(self):
         )
         self.assertTrue(context.is_valid)
 
-    def test_is_valid_with_valid_web_context_returns_true(self):
+    def test_is_valid_with_valid_web_context_returns_true(self) -> None:
         context = parameter_domain.EvaluationContext.from_dict(
             {
                 'platform_type': 'Web',
                 'browser_type': 'Chrome',
+                'app_version': None,
```

Comment:
After defining a `TypedDict` ( ClientSideContextDict ) for a dictionary, all keys of that dictionary should be present. Also, in `platform_parameter_domain.py` file we are using `.get()` method, so even if we don't provide the `app_version` in the dictionary its value is eventually counted as `None`.

Reply:
If we do not add `app_version` here then MyPy test fails, because after adding annotations the first argument of `EvaluationContext.from_dict()` method can only accept `ClientSideContextDict` and in `ClientSideContextDict`, `app_version`,  `browser_type` and `platform_type` key is defined.